src/main/java/com/epam/lab/paymentsystem/dao/ConnectionPool.java

```diff
@@ -41,36 +43,30 @@ public static void connectionRelease(Connection connection) {
                 e.printStackTrace();
             }
             connectionPool.add(con);
+            LOGGER.info("Connection returned back to the database");
         }
     }
 
     private static void addConnect() {
         for (int i = 1; i <= CONNECTION_POOL_SIZE; i++) {
             connectionPool.add(new ConnectionPool().connect);
         }
+        LOGGER.info("Connection pool created, amount connections: " + connectionPool.size());
     }
 
     public ConnectionPool() {
         try {
             Class.forName(DRIVER_DATABASE_CLASS);
         } catch (ClassNotFoundException e) {
+            LOGGER.error("Exception, PostgreSQL JDBC Driver is not registered" + e);
             e.printStackTrace();
```

Comment:
I think `LOGGER.error(message, e)` is enough, it prints stack trace as well so `e.printStackTrace()` is redundant. Can you check it out?